src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/RuntimeInformation.Unix.cs

```diff
@@ -10,7 +10,24 @@ public static partial class RuntimeInformation
         private static string? s_osDescription;
         private static volatile int s_osArchPlusOne;
 
-        public static string OSDescription => s_osDescription ??= (GetPrettyOSDescription() ?? Interop.Sys.GetUnixVersion());
+        public static string OSDescription => s_osDescription ??=
+#if TARGET_ANDROID
+            $"Android (API {Environment.OSVersion.Version.Major})";
```

Comment:
```suggestion
            $"Android (API level {Environment.OSVersion.Version.Major})";
```
As far as I can tell, this version is typically referred to as "API level" not just "API", for example https://developer.android.com/tools/releases/platforms . Would it look better here?

Reply:
done